package.json

```diff
@@ -2,7 +2,7 @@
     "name": "@nathanstitt/sundry",
     "private": false,
     "sideEffects": false,
-    "version": "0.0.42",
+    "version": "0.0.43",
```

Comment:
just so ya know, I use https://www.npmjs.com/package/np to publish packages and don't need to manually bump the version.

Reply:
oh cool!